COPYRIGHT

```diff
@@ -1 +1 @@
-Copyright (c) <2019> <Samsung Inner Source Community> All Rights Reserved.
+Copyright (c) 2019 Samsung Electronics Co., Ltd. All Rights Reserved.
```

Comment:
@MoonkiHong , reflected your opinion. :)

```suggestion
Copyright (c) 2019 Samsung Electronics Co., Ltd. & Contributors All Rights Reserved.
```

Reply:
Agree. Thank you for the kind suggestion and revision. :)

Reply:
@MoonkiHong , if you agree to this suggestion, you need to push `commit suggestion` to apply. :)